NSS-Day-Cohort-42/holiday-road-wailing-worms
# Cj attractions component

# Steps to test

1. `git fetch --all`
1. `git checkout cj-attractions`
1. Confirm that the dropdown box appears on page
1. Confirm that you are able to select an attraction
1. Confirm that you are then able to see a modal displaying more details about the attraction after the details button is clicked
